addons/website/static/src/snippets/s_rating/000.scss

```diff
@@ -49,7 +49,7 @@
             }
         }
     }
-    > .s_rating_1x { .fa { font-size: 1em; }; }
-    > .s_rating_2x { .fa { font-size: 2em; }; }
-    > .s_rating_3x { .fa { font-size: 3em; }; }
+    > .s_rating_1x { .fa { @include font-size(1em); }; }
+    > .s_rating_2x { .fa { @include font-size(2em); }; }
+    > .s_rating_3x { .fa { @include font-size(3em); }; }
```

Comment:
I am curious, how does the mixin can work with `em` values? From what I remember responsive font-size of bootstrap work like this:

![](https://raw.githubusercontent.com/twbs/rfs/master/.github/rfs-graph.svg?sanitize=true)

But then with `em` values this would depend on the parent, is there some sort of magic or does `@include font-size(2em);` === `font-size: 2em;`